pallets/subtensor/src/math.rs

```diff
@@ -206,6 +206,7 @@ pub fn sigmoid_safe(input: I32F32, rho: I32F32, kappa: I32F32) -> I32F32 {
 
 // Returns a bool vector where an item is true if the vector item is in topk values.
 #[allow(dead_code, clippy::indexing_slicing)]
+#[test_fuzz::test_fuzz]
```

Comment:
Is it possible to add bounds to these?  My fear is that the fuzzer would run iterations on values outside what is realistic.

Reply:
I mean that's probably a feature rather than a bug unless it causes a false positive someday